metadata-ingestion/src/datahub/ingestion/source/mode.py

```diff
@@ -169,12 +182,21 @@ def _get_ownership(self, user: str) -> Optional[OwnershipClass]:
 
     @lru_cache(maxsize=None)
     def _get_creator(self, href: str) -> str:
-        user = self.session.get(f"{self.config.connect_uri}{href}")
-        user_json = user.json()
-        if self.config.owner_username_instead_of_email:
-            return user_json.get("username", "unknown")
-        else:
-            return user_json.get("email", "unknown")
+        user = ""
+        try:
+            user_json = self._get_request_json(f"{self.config.connect_uri}{href}")
+            user = (
+                user_json.get("username", "unknown")
```

Comment:
Nit: if we don't get a username, i'd rather we don't say anything as opposed to putting some "unknown" stuff in the DB

Reply:
fixed by [commit 62afea1237a57c54b65845b1fe0a313a037f7c88](https://github.com/linkedin/datahub/pull/3761/commits/62afea1237a57c54b65845b1fe0a313a037f7c88).

creating an ChangeAuditStamps does create an URN with unknown name though.
`ChangeAuditStampsClass({'created': AuditStampClass({'time': 0, 'actor': 'urn:li:corpuser:unknown', 'impersonator': None}), 'lastModified': AuditStampClass({'time': 0, 'actor': 'urn:li:corpuser:unknown', 'impersonator': None}), 'deleted': None})`